ddtrace/contrib/internal/aiobotocore/patch.py

```diff
@@ -116,7 +117,7 @@ async def _wrapped_api_call(original_func, instance, args, kwargs):
         schematize_cloud_api_operation(
             "{}.command".format(endpoint_name), cloud_provider="aws", cloud_service=endpoint_name
         ),
-        service=schematize_service_name(service),
+        service=ext_service(None, config.aiobotocore, default=schematize_service_name(service)),
```

Comment:
So the aiobotocore logic for service name is messy currently, these changes were necessary to ensure the same tests pass for schema version v1 without any changes, which would be breaking. We don't include the pin within this call since we would always default to the `pin.service` which in this case would be `aws.[service]`. However, for current V1 behavior, aiobotocore defaults to the default service name, this can be confirmed by looking at the current test cases.

Reply:
Fixed